systemtest/010-inspect.bats

```diff
@@ -27,11 +27,19 @@ load helpers
     # Now run inspect locally
     run_skopeo inspect dir:$workdir
     inspect_local=$output
-
-    # Each SHA-named file must be listed in the output of 'inspect'
+    run_skopeo inspect --raw dir:$workdir
+    inspect_local_raw=$output
+    config_digest=$(jq -r '.config.digest' <<<"$inspect_local_raw")
+
+    # Each SHA-named layer file (but not the config) must be listed in the output of 'inspect'.
+    # As of Skopeo 1.6, (skopeo inspect)'s output lists layer digests,
+    # but not the digest of the config blob ($config_digest), if any.
```

Comment:
Non-blocking: This is the 1.4 branch, so the comment is slightly confusing.  Is it saying "Prior to 1.6, the output also included the digest of the config blob"?

Reply:
It is meant to say “as of all existing versions to date, up to and including 1.6”.